build.sbt

```diff
@@ -39,15 +39,26 @@ lazy val root =
       lambdaEvents.js,
       lambdaEvents.jvm,
       lambdaApiGatewayProxyHttp4s.js,
-      lambdaApiGatewayProxyHttp4s.jvm)
+      lambdaApiGatewayProxyHttp4s.jvm,
+      cloudflareWorker
```

Comment:
Is there any reason not to add `core.jvm` and `core.js` to this list? It makes testing things locally a little easier and we may need to anyway to make sure those artifacts get published.

Reply:
Woops, just forgot to! Good catch :)